samples/AspNetCore/ControllerSample/Startup.cs

```diff
@@ -37,13 +37,13 @@ public Startup(IConfiguration configuration)
         /// <param name="services">Service Collection.</param>
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddControllers().AddDapr();
-
-            services.AddSingleton(new JsonSerializerOptions()
-            {
-                PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
-                PropertyNameCaseInsensitive = true,
-            });
+            services.AddControllers().AddDapr(builder => 
```

Comment:
We can remove the customized JSOn settings from the startup now that defaults are good enough.